deploy/HTTPS-SETUP.md

```
@@ -0,0 +1,35 @@
+  -d your-domain.com -d www.your-domain.com \
+  --email your-email@example.com --agree-tos --no-eff-email
+```
+
+## 3) Switch nginx to HTTPS config
+```bash
+cp nginx/nginx.https.template.conf nginx/nginx.https.conf
+sed -i "s/__DOMAIN__/your-domain.com/g" nginx/nginx.https.conf
+cp nginx/nginx.https.conf nginx/nginx.conf
+```
+
+## 4) Reload nginx
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)